keps/sig-network/1669-graceful-termination-local-external-traffic-policy/README.md

```diff
@@ -232,22 +228,22 @@ checking if there are objects with field X set) may be a last resort. Avoid
 logs or events for this purpose.
 -->
 
-TBD for beta.
+* A Service Type=LoadBalancer sets externalTrafficPolicy=Local.
+* The load balancer implementation uses `spec.healthCheckNodePort` for node health checking.
+* A Pod can receive traffic during termination.
```

Comment:
Those conditions are non-trivial to determine for cluster operators.

How about having the metrics I suggested above [this won't allow the answer for specific service, but at least for conjunctions of all services.